src/components/SocialLink/package.json

```diff
@@ -8,6 +8,6 @@
     "prop-types": "^15.5.10",
     "react": "^15.6.1",
     "react-svg": "^2.1.22",
-    "@comicrelief/pattern-lab": "^7.4.1"
+    "@comicrelief/pattern-lab": "^7.63.0"
```

Comment:
Looking good! Let's just update this to use a _specific_ PL version 👍

Reply:
i.e. "7.63.0", and updating the lockfile tooooo

Reply:
Oh man I'm an idiot, THIS one _should_ be `'^7.63.0'` as it's a dependency of the component itself. If we demand a specific version here like I suggested, we'll get dependency issues in any app that's using a more recent version of PL.

Sorry dude! We can put this back to how it was, sorry!